tt_metal/impl/dispatch/command_queue.hpp

```diff
@@ -306,12 +309,6 @@ class EnqueueWrapCommand : public Command {
     EnqueueCommandType type() { return EnqueueCommandType::ENQUEUE_WRAP; }
 };
 
-// Fwd declares
-namespace detail{
-    CommandQueue &GetCommandQueue(Device *device);
-}
-
-
 class Trace {
```

Comment:
May I ask what is the role of Trace objects?

Reply:
`Trace` is @DrJessop's new mechanism for batching up multiple programs for better runtime performance